apps/classroomio-com/src/routes/tools/progress-report/components/Report.svelte

```diff
@@ -0,0 +1,141 @@
+<script lang="ts">
+  import { OpenFullscreen, htmlBody, toggleFullscreenModal, nodeStore } from './store';
+
+  let node: any;
+
+  $: nodeStore.set(node);
+</script>
+
+<div class="md:h-[50vh] w-full border relative" bind:this={node}>
+  <!-- fullscreen button -->
+  {#if $OpenFullscreen.open}
+    <button
+      type="button"
+      on:click={toggleFullscreenModal}
+      class="absolute top-4 right-4 p-1.5 rounded-full hover:scale-90 transition-all duration-300 bg-white"
+    >
+      <img src="/free-tools/progress-report/close-icon.svg" alt="" class="" />
+    </button>
+  {:else}
+    <button
+      type="button"
+      on:click={toggleFullscreenModal}
+      class="absolute top-4 right-4 w-7 hover:scale-90 transition-all duration-300"
+    >
+      <img src="/free-tools/progress-report/full-screen-icon.svg" alt="" class="" />
+    </button>
+  {/if}
+
+  <!-- background image -->
+  {#if $htmlBody.background}
+    <img
+      src="https://assets.cdn.clsrio.com/progress-report/backgrounds/{$htmlBody.background}.png"
+      alt=""
+      class="absoute z-10 h-full w-full object-cover"
+    />
+  {:else}
+    <img
+      src="https://assets.cdn.clsrio.com/progress-report/backgrounds/blue_tetiary_background.png"
+      alt=""
+      class="absoute z-10 h-full w-full object-cover"
+    />
+  {/if}
```

Comment:
Lol but why do you use absolute a lot for things that can be semantically styled.

If this is a background image, why can't it just be something like this:

<img width="999" alt="image" src="https://github.com/rotimi-best/classroomio/assets/31730715/b5eec80c-3a1f-4979-83ef-91da93027dd7">

Just add the background image to the element rather than using a ton of absolute and positions. This approach will make it hard to improve the styling